x-pack/legacy/plugins/apm/server/lib/transaction_groups/fetcher.ts

```diff
@@ -63,6 +63,7 @@ export function transactionGroupsFetcher(
         transactions: {
           terms: {
             ...projection.body.aggs.transactions.terms,
+            execution_hint: 'map',
```

Comment:
What field are we aggregating on here?

Reply:
hmm.. it looks like `transaction.name`. That's not necessarily a high cardinality field.